extracts/v_extracts.nj_position_control_roster.sql

```diff
@@ -0,0 +1,49 @@
+USE gabby
+GO
+
+CREATE OR ALTER VIEW extracts.nj_position_control_roster AS
+
+--CTE for only titles and entities needed (including all NJ staff and CEO and CFO from KTAF)--
+WITH nj_reportable AS (
+    SELECT
+       employee_number
+      ,CASE 
+       WHEN sr.job_title = 'Chief Financial Officer' THEN '1'
```

Comment:
mixing strings and ints: does this not error out?

Reply:
it didn't, but I don't need this anymore so all good.